x-pack/platform/plugins/private/data_visualizer/public/application/file_data_visualizer/components/analysis_summary/analysis_summary.tsx

```diff
@@ -14,29 +14,37 @@ import type { FindFileStructureResponse } from '@kbn/file-upload-plugin/common';
 import { getTikaDisplayType } from '../../../../../common/utils/tika_utils';
 import { FILE_FORMATS } from '../../../../../common/constants';
 
-export const AnalysisSummary: FC<{ results: FindFileStructureResponse }> = ({ results }) => {
+interface Props {
+  results: FindFileStructureResponse;
+  showTitle?: boolean;
+}
+
+export const AnalysisSummary: FC<Props> = ({ results, showTitle = true }) => {
   const items = createDisplayItems(results);
 
   return (
-    <React.Fragment>
-      <EuiTitle size="s">
-        <h2>
-          <FormattedMessage
-            id="xpack.dataVisualizer.file.analysisSummary.summaryTitle"
-            defaultMessage="Summary"
-          />
-        </h2>
-      </EuiTitle>
+    <>
+      {showTitle ? (
+        <EuiTitle size="s">
+          <h2>
+            <FormattedMessage
+              id="xpack.dataVisualizer.file.analysisSummary.summaryTitle"
+              defaultMessage="Summary"
+            />
+          </h2>
+        </EuiTitle>
+      ) : null}
 
       <EuiSpacer size="m" />
 
       <EuiDescriptionList
         type="column"
-        columnWidths={[15, 85]}
+        columnWidths={[30, 70]}
```

Comment:
Could this go to `50, 50` to reduce the likelihood the text of the first row wraps? The lengths of the number of lines / file type field values should always be fairly short shouldn't they?

<img width="496" alt="Screenshot 2025-03-06 at 13 15 06" src="https://github.com/user-attachments/assets/d64ef1c8-8e9e-402e-8d01-465ccabb038f" />

Reply:
Updated in https://github.com/elastic/kibana/pull/213275/commits/75af43ec0d9cd50753992b2d312f760ce81ea148
![image](https://github.com/user-attachments/assets/b8d1d64c-13b4-4498-b913-0fdd8ba9f5c6)